aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -542,18 +533,6 @@ module aptos_token::token {
         deposit_token(receiver, token);
     }
 
-    public fun initialize_token(account: &signer, token_id: TokenId) acquires TokenStore {
```

Comment:
please deprecate this function in the same way as the one above

Reply:
Updated!